Signal/src/ViewControllers/ConversationView/ConversationInputToolbar.m

```diff
@@ -159,51 +121,60 @@ - (void)createContents
     [self.attachmentButton addTarget:self
                               action:@selector(attachmentButtonPressed)
                     forControlEvents:UIControlEventTouchUpInside];
-    [self.attachmentButton setImage:[UIImage imageNamed:@"btnAttachments--blue"] forState:UIControlStateNormal];
-    self.attachmentButton.contentEdgeInsets = UIEdgeInsetsMake(0, 3, 0, 3);
-    [self.leftButtonWrapper addSubview:self.attachmentButton];
+    UIImage *attachmentImage = [UIImage imageNamed:@"btnAttachments--blue"];
+    [self.attachmentButton setImage:[attachmentImage imageWithRenderingMode:UIImageRenderingModeAlwaysTemplate]
+                           forState:UIControlStateNormal];
+    self.attachmentButton.tintColor = UIColor.ows_navbarIconColor;
+    [self.attachmentButton autoSetDimensionsToSize:CGSizeMake(40, kMinTextViewHeight)];
```

Comment:
To increase "hit area" rather than wrapping our views as sizing the wrapper, we can simply size the button.